### ISSUE TITLE
TrueOS-Desktop-2016-08-31-x64-USB.img boot hang waiting for rtsol

### ISSUE THREAD

**Issue by hxw** [trueos/trueos-core#43]
After install and reboot the startup scripts hang after displaying:

```
wlan0: no link .... giving up
/etc/rc.d/dhclient: WARNING: failed to start dhclient
iwn0: radio is disabled by hardware switch
```
`Ctrl-T` message shows rtsol in state select; switching on the radio has no effect at this point and requires `Ctrl-C` to continue the startup.

**Comment by pkgdemon** [comment]
The man page for iwn does mention this error.  

https://www.freebsd.org/cgi/man.cgi?query=iwn&apropos=0&sektion=0&manpath=OpenBSD+4.7&format=html

Can you verify the hardware switch is indeed set to enable the wireless adapter?  Do you have a bios option for disabling the switch?

**Comment by hxw** [comment]
Normally the switch is off, as the notebook is using wired Ethernet most of the time and I only turn it on if I take the machine somewhere that is WiFi only.

Once I have removed the `accept_rtadv` from the wlan0 config there is no hang, so based on your OpenBSD man page I suppose rtsol is stuck waiting to send.  I did not see any similar message in FreeBSD `man iwn` page.

From Lumina desktop, the  *Wireless Configuration* dialog scan shows nothing when switch is off but needs two clicks of scan button to actually get list.  Command line is the same, after switch on need to run: `ifconfig wlan0 scan ; ifconfig wlan0 scan` to get the listing. (no delay necessary between commands)

I did a quick check with `rtsol -dD wlan0` switch off, then turned on.  rtsol only sees network down condition when the switch is on and only then will rtsol time out.

Seems more like an issue with FreeBSD iwn driver not setting "network down" state when switch is off, so maybe just close this.
